web/src/components/Collapsible.tsx

```diff
@@ -7,10 +7,10 @@ import { helpers } from '../constants/theme'
 import Button from './base/Button'
 import Icon from './base/Icon'
 
-const Container = styled.div`
+const Container = styled.div<{ $isGapEnabled: boolean }>`
```

Comment:
🔧 I think this is not the best solution sadly :/ Introducing a separate prop for one style only is quite complicated. Instead, it would be better to just revert the gap here and overwrite the style where you use the collapsible and do not want the gap. You can achieve this as follows:
- Add a new prop `className` and pass it to this styled component `Container` in L57
-  Overwrite the style in web/src/PoiDetails as follows:
``` ts
const StyledCollapsible = styled(Collapsible)`
  gap: 0;
`
```